pgmq-extension/Makefile

```diff
@@ -39,7 +40,7 @@ Trunk.toml:
 install-pg-partman:
 	git clone https://github.com/pgpartman/pg_partman.git && \
 	cd pg_partman && \
-	git checkout v5.1.0 && \
+	git checkout v${PG_PARTMAN_VER} && \
```

Comment:
This is just so it can be successfully installed, yes? Doesn't imply that this version has to be installed, because pgmq will work with either version, correct?

Reply:
Correct. It runs on both 4.7 and 5.1. This is just a helper function for installing pg_partman. This particular change is just so we can configure which version is installed in CI.

Reply:
and we wouldnt need to make this change if pg partman 5.1.0 were already in pgxn